tool/tbot/testdata/TestRun_Configure/all_parameters_provided/file.golden

```diff
@@ -2,13 +2,13 @@
 onboarding:
   token: xxyzz
   ca_pins:
-  - sha256:capindata
+    - sha256:capindata
```

Comment:
Switching to the V3 version of the YAML library has introduced these changes to the formatting of the YAML.